lib/templates/spregen-config.yml.erb

```diff
@@ -3,9 +3,11 @@
 # Example: https://trello.com/1/authorize?key=YOURAPIKEY&response_type=token&expiration=never
 # You should see a page asking you to authorize your Trello application. Click "allow" and you should see a second page with a long alphanumeric string. This is your member token.
 
+# This is just a template. Copy it to /etc/spregen/config.yml
```

Comment:
This template is copied to ~/.spregen-config.yml and this message are still displayed, even when it is not a template anymore